akka-stream/src/main/boilerplate/akka/japi/tuple/Tuple.scala.template

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.japi.tuple
+
+[3..22#/**
+ * Used to create tuples in Java.
+ */
+object Tuple1 {
+  def create[[#T1#]]([#t1: T1#]) = new Tuple1([#t1#])
+}
+
+/**
+ * Java API Tuple container.
+ */
+@SerialVersionUID(##1L)
+case class Tuple1[[#T1#]]([#val t1: T1#]) extends scala.AnyRef with scala.Product with scala.Serializable {
```

Comment:
If `case class` then please add `final` and drop the `val` from `t1`, let's use what it provides.
Is there a reason to manually extend `AnyRef`?